_data/sites.json

```diff
@@ -8299,6 +8299,16 @@
         ]
     },
 
+    {
+        "name": "WT.Social",
+        "url": "https://wt.social/myaccount/deactivate-account",
+        "difficulty": "easy",
+        "notes": "Only deactivation is possible",
+        "domains": [
+            "wt.social"
+        ]
+    },
```

Comment:
The whole entry should be placed after `workupload` and before `Wunderlist` to keep the file sorted.

Reply:
Sry - I thought "popular" was the name of the next entry